data_steward/utils/concept_ids_suppression.py

```diff
@@ -0,0 +1,67 @@
+"""
```

Comment:
This script is more similar to the scripts that we have in `data_steward/tools` which are mostly one-off or the scripts we need to run our cdr generation pipeline. `data_steward/utils` is being used to store the scripts that wrap functions in a third-party library to our project needs. let's maintain the same organization.